.codegen/service.py.tmpl

```diff
@@ -338,6 +370,22 @@ self._api.do('{{.Verb}}',
   {{- end}}
   {{- if and .IsRequestByteStream .RequestBodyField }}, data={{template "safe-snake-name" .RequestBodyField}}{{ end }}
   {{- if .IsResponseByteStream }}, raw=True{{ end }})
+{{- else -}}
+  self._api.do('{{.Verb}}',
+    url=data_plane_details.endpoint_url
```

Comment:
Small suggestion: I think we don't need to duplicate all of the parts of how we call the `do()` method here, since the only differences are in the URL and auth we supply. So something like
```
{{ if .Service.IsDataPlane }}
, url=data_plane_details.endpoint_url
, auth=auth
{{ else }}
{{ template "path" . }}
{{ end }}
```
would reduce duplication in this template.